contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -876,6 +876,20 @@ kind `Foo` by just the name (within the current namespace, if a namespaced
 resource), or should be called `fooRef`, and should contain a subset of the
 fields of the `ObjectReference` type.
 
+Object references on a namespaced type should usually refer only to objects in
+the same namespace.  Because namespaces are a security boundary, cross namespace
```

Comment:
If we're going to be squishy with a "usually", should we be clearer in saying built-in types and ownerReferences don't support cross-namespace references, and if a non-built-in type chooses to, it should do double opt-in or permission checks to prevent these problems?

Is it worth explicitly saying here that object references on built-in kubernetes types and in ownerReferences do not support cross namespace references

Reply:
> If we're going to be squishy with a "usually", should we be clearer in saying built-in types and ownerReferences don't support cross-namespace references, and if a non-built-in type chooses to, it should do double opt-in or permission checks to prevent these problems?
> 
> Is it worth explicitly saying here that object references on built-in kubernetes types and in ownerReferences do not support cross namespace references

updated